src/panels/lovelace/cards/hui-markdown-card.ts

```diff
@@ -41,7 +41,7 @@ export class HuiMarkdownCard extends LitElement implements LovelaceCard {
 
   @property() private _content = "";
 
-  @property() private _unsubRenderTemplate?: Promise<UnsubscribeFunc>;
+  @property() private _unsubRenderTemplate?: UnsubscribeFunc;
```

Comment:
The reason we stored it as a template was that we would guarantee to clean it up if the element was thrown away before the subscription would resolve to an unsubscribe function.